src/mainmenu.h

```diff
@@ -29,6 +29,8 @@ class MainMenu: public GameState
 	Vector2 StartDims;
 	int TitleFontSize;
 
+	Vector2 VersionPos;
```

Comment:
Are these variables actually used? The version number text should just be part of the title screen

Reply:
forgot to remove that